src/obfl-specification.html

```diff
@@ -1606,6 +1606,62 @@ <h4 id="L1105" class="include">toc-entry</h4>
 </dl>
 </div>
 
+<h4 id="L1105" class="include">toc-entry-on-resumed</h4>
+
+<p>The toc-entry-on-resumed element defines an entry in a table of contents. It is a
+conditional container of content for volume breaks. The contents of at most one
+toc-entry-on-resumed per volume will be rendered at the start of the resumed volume's
+table of contents.
+This functionality is only available in toc-sequences with a volume-resumed or
+document-resumed range. When there are several volume breaks in the range of a
+toc-entry-on-resumed, its contents can be rendered more than once.
+The range consists of two block ids (the latter is optional) in a notation similar to  
+<a href="https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402">Maven
+version ranges</a> that refer to the start of a block. It is required that:
+<ul>
+<li>
+   Ranges become smaller when you descend in TOC tree. More formally, if there are
+   toc-entry-on-resumed elements A and B such that the toc-block of A
+   is an <a href="https://www.w3.org/TR/2017/REC-xpath-31-20170321/#prod-xpath31-ReverseAxis">ancestor</a>
+   of the toc-block of B, then every block in B's range must be in A's range.
+</li>
+<li>
+   The order of the toc-entry-on-resumed elements is consistent with the
+   blocks they reference.
+</li>
+</ul>
+When a volume break occurs, the toc-entry-on-resumed is selected that is deepest in the
+TOC tree. The above requirements guarantee that there is at most one. 
+</p>
+
+<div class="sidebar markup">
+<dl>
+  <dt>Usage</dt>
+    <dd>toc-block</dd>
+  <dt>Attributes</dt>
+    <dd><dl>
+        <dt>id-range [required]</dt>
```

Comment:
I was thinking about this name, "id-range", the other day. The name suggests that it refers to a range of IDs, but it doesn't. It's a range of blocks, of which the first and last have the specified ID. I believe @fredrikschill 's intention was to make the name more alike the existing attribute`ref-id`, but I'm not so sure anymore if that goal is achieved this way.

Reply:
You have a point in this @bertfrees . We can stick to your initial suggestion with just `range` or can you come up with anything more descriptive?

Reply:
I have already thought about a more descriptive name but haven't come up with one yet.

Reply:
I will change it to "range". Please feel free to suggest a better name.

Reply:
:+1: